frontend/src/features/admin-form/responses/FeedbackPage/FeedbackTable.tsx

```diff
@@ -0,0 +1,188 @@
+import { useEffect, useMemo } from 'react'
+import {
+  Column,
+  useFlexLayout,
+  usePagination,
+  useResizeColumns,
+  useSortBy,
+  useTable,
+} from 'react-table'
+import { Flex, Icon, Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react'
+
+import { ProcessedFeedbackMeta } from '~shared/types/form'
+
+import { BxsChevronDown } from '~assets/icons/BxsChevronDown'
+import { BxsChevronUp } from '~assets/icons/BxsChevronUp'
+
+type FeedbackColumnData = {
+  index: number
+  date: string
+  feedback: string
+  rating: number
+}
+
+const FEEDBACK_TABLE_COLUMNS: Column<FeedbackColumnData>[] = [
+  {
+    Header: '#',
+    accessor: 'index',
+    sortType: 'basic',
+    minWidth: 80, // minWidth is only used as a limit for resizing
+    width: 80, // width is used for both the flex-basis and flex-grow
+    maxWidth: 100, // maxWidth is only used as a limit for resizing
+  },
+  {
+    Header: 'Date',
+    accessor: 'date',
+    sortType: (rowA, rowB, columnId) => {
+      const dateA = new Date(rowA.values[columnId]) //rowA.values[columnId] is a date string
+      const dateB = new Date(rowB.values[columnId])
+      return dateA > dateB ? 1 : -1
+    },
```

Comment:
Use `timestamp` as accessor and compare with a plain difference a - b . 

And reformat date in frontend.

Reply:
6eb78c1